src/apps/Map/index.tsx

```diff
@@ -145,16 +163,26 @@ const MapView = ({
             />
           )}
           <StyledMapControl position="bottom-right" role="button">
-            {!showLegend && (
-              <StyledMapLegendButton
-                onClick={() => setShowLegend(!showLegend)}
-              />
-            )}
-            {showLegend && (
-              <StyledMapLegendButtonActivated
-                onClick={() => setShowLegend(!showLegend)}
-              />
-            )}
+            <table>
+              <tr>
+                  <StyledZoomInButton onClick={() => changeZoom(1)} />
+              </tr>
+              <tr>
+                  <StyledZoomOutButton onClick={() => changeZoom(-1)} />
+              </tr>
+              <tr>
+                  {!showLegend && (
+                    <StyledMapLegendButton
+                      onClick={() => setShowLegend(!showLegend)}
+                    />
+                  )}
+                  {showLegend && (
+                    <StyledMapLegendButtonActivated
+                      onClick={() => setShowLegend(!showLegend)}
+                    />
+                  )}
+              </tr>
+            </table>
```

Comment:
I would prefer using [Flexbox](https://developer.mozilla.org/de/docs/Learn/CSS/CSS_layout/Flexbox) instead of a table, see also [CSS Tricks](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).